engine/apps/slack/models/slack_message.py

```diff
@@ -68,12 +68,8 @@ def slack_team_identity(self):
                     f"It is strange!"
                 )
                 return None
-            # Re-take object to switch connection from readonly db to master.
             self._slack_team_identity = self.organization.slack_team_identity
-
-            _self = SlackMessage.objects.get(pk=self.pk)
-            _self._slack_team_identity = _self.organization.slack_team_identity
-            _self.save()
```

Comment:
AFAIK, we don't make any usage of read-only databases atm, so there should be no need to refetch `self`